### PR TITLE
Add Winter 2021 events to website

### PR DESCRIPTION
**Pull request by aays**

### CODE REVIEW

File: _data/events.yml [hunk 1]
```diff
@@ -23,19 +23,47 @@
   # R or Python
   program_language: Any
```
Comment (QuLogic) [inline _data/events.yml]: Shouldn't this by Python?

File: _data/events.yml [hunk 2]
```diff
@@ -23,19 +23,47 @@
...
+  # The URL for the location
+  location_url: ""
+  # Packages to install, a space separating each package
+  # (e.g. "dplyr tidyr" or "Pandas>=0.21 Seaborn")
+  packages: ""
+  youtube_link: ""
+  skill_level: Beginner
+  # Beginner Intermediate Advanced
+  # R or Python
+  program_language: Any
```
Comment (QuLogic) [inline _data/events.yml]: And this one R?

File: _data/events.yml [hunk 3]
```diff
@@ -47,22 +75,48 @@
...
+  # Start and end time for event in 24 hour HH:MM format
+  start_time: "18:00"
+  end_time: "19:00"
+  # Where the event will take place
+  location: Zoom
+  # The URL for the location
+  location_url: ""
+  # Packages to install, a space separating each package
+  # (e.g. "dplyr tidyr" or "Pandas>=0.21 Seaborn")
+  packages: ""
```
Comment (QuLogic) [inline _data/events.yml]: scikit-learn, at least, I guess.

File: _posts/2021-01-13-intro-python.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Introduction to Python"
+text: "A brief introduction to Python for beginners.
+"
```
Comment (QuLogic) [inline _posts/2021-01-13-intro-python.md:4]: Has it always been adding this extra newline? Does that actually work?

Comment (aays) [reply _posts/2021-01-13-intro-python.md:4]: I've just worked off of previous formatting (which included this) but I could remove it and see what happens

File: _data/events.yml (hunk 3 above)
Comment (aays) [reply _data/events.yml]: Whoops, good catch!

File: _data/events.yml (hunk 2 above)
Comment (aays) [reply _data/events.yml]: Good catch, this is what I get for leaving putting this together for the last task of a long day... on it

File: _posts/2021-01-13-intro-python.md (hunk 4 above)
Comment (aays) [reply _posts/2021-01-13-intro-python.md:4]: Wait, I didn't realize this wasn't `events.yml` - yeah, it seems the newline is being automatically added. Weird

**Pull request merged**